db/migrate/20181018082620_create_provided_access_tokens.rb

```diff
@@ -0,0 +1,11 @@
+class CreateProvidedAccessTokens < ActiveRecord::Migration
+  def change
+    create_table :provided_access_tokens do |t|
+      t.text :value
```

Comment:
In case we need/can use in the future, do we add a `refresh_token` column? Or would a `User` have two tokens (one of them being the refresh one)?

Reply:
We can add it in this table as another column.
Lets wait :)